lib/fastlane/plugin/datadog/actions/upload_symbols_to_datadog.rb

```diff
@@ -48,20 +49,35 @@ def self.details
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :api_key,
-          env_name: 'DATADOG_API_KEY',
-          description: "Datadog API Key for upload_symbols_to_datadog",
-          verify_block: proc do |value|
-            UI.user_error!("No API key for upload_symbols_to_datadog given, pass using `api_key: 'api_key'`") unless value && !value.empty?
-          end),
-          FastlaneCore::ConfigItem.new(key: :dsym_paths,
-          default_value: Actions.lane_context[SharedValues::DSYM_PATHS],
-          description: "An array of the folders and/or the zip files which contains the dSYM files",
-          type: Array),
-          FastlaneCore::ConfigItem.new(key: :dry_run,
-          description: "No upload to Datadog",
-          default_value: false,
-          is_string: false)
+          FastlaneCore::ConfigItem.new(
+            key: :api_key,
+            env_name: 'DATADOG_API_KEY',
+            description: "Datadog API Key for upload_symbols_to_datadog",
+            verify_block: proc do |value|
+              UI.user_error!("No API key for upload_symbols_to_datadog given, pass using `api_key: 'api_key'`") unless value && !value.empty?
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :site,
+            env_name: 'DATADOG_SITE',
+            default_value: 'datadoghq.com',
+            description: "Datadog site region. `datadoghq.com` by default, use `datadoghq.eu` for the EU",
+            verify_block: proc do |value|
+              UI.user_error!("`site` parameter should be 'datadoghq.com' or 'datadoghq.eu'") unless value == 'datadoghq.com' || value == 'datadoghq.eu'
+            end
```

Comment:
Should we be more flexible here?
We could remove any verification and let the cmd fail, so we won't have to update this plugin when new sites are added to `datadog-ci`.

Reply:
my gut feeling is that verification might cause more problems than it solves. i don't think we need this verification at all.
this site is logged to Terminal by `datadog-ci` tool, our users should be able to see if they have something wrong there